packages/services/api/src/modules/auth/lib/authz.ts

```diff
@@ -185,7 +185,7 @@ export abstract class Session {
               args.organizationId,
               args.params,
             );
-            throw new AccessError(`Missing permission for performing '${args.action}' on resource`);
+            throw new InsufficientPermissionError(args.action);
```

Comment:
:+1: make sure to update the documentation if we're replacing this error with another.

Reply:
It is still the same error, this just allows to instantiate it somewhere else